onnxruntime/python/tools/quantization/quant_utils.py

```diff
@@ -296,11 +296,25 @@ def generate_identified_filename(filename: Path, identifier: str) -> Path:
     '''
     return filename.parent.joinpath(filename.stem + identifier).with_suffix(filename.suffix)
 
+def check_and_install_module(module):
```

Comment:
>check_and_install_module [](start = 4, length = 24)

does it work to add it to [https://github.com/microsoft/onnxruntime/blob/72eb5de0e21a140371ab07222c0f43faff94e927/requirements.txt](https://github.com/microsoft/onnxruntime/blob/72eb5de0e21a140371ab07222c0f43faff94e927/requirements.txt)

Reply:
I've tested it and it works.